### PR TITLE
Feat e2e signout

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/556/mobile-tests-sign-out
#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: test/spec/logout.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: Logout
+
+    Scenario: User logs out successfully
+        Given I have signed up
+        When I go to settings tab
+        Then I press the sign out button
+        And I confirm sign out
```
Comment (alexandra-marin) [inline test/spec/logout.feature]: 'I go to settings tab', 'I press the sign out button' and 'I confirm sign out' are implementation details. You can combine them into a single step, 'I sign out'.

File: test/spec/logout.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Feature: Logout
+
+    Scenario: User logs out successfully
+        Given I have signed up
```
Comment (alexandra-marin) [inline test/spec/logout.feature]: 'Given I have signed up' can be moved in the Background section

File: test/spec/logout.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        When I go to settings tab
+        Then I press the sign out button
+        And I confirm sign out
+        And I close Peerio
+        When I open Peerio
+        Then I am taken to the welcome screen
```
Comment (alexandra-marin) [inline test/spec/logout.feature]: Welcome screen should be shown to logged in users. After they've signed out, it should be Login/Create account screen.

File: test/spec/logout.feature (hunk 2 above)
Comment (Karim94) [reply test/spec/logout.feature]: Done

File: test/spec/logout.feature (hunk 1 above)
Comment (Karim94) [reply test/spec/logout.feature]: Done

File: test/spec/logout.feature (hunk 3 above)
Comment (Karim94) [reply test/spec/logout.feature]: Renamed. I meant it to be the same thing, I just chose an unsuitable name.

**Comment by Karim94** [comment]
Cherry picked commits and moved to new branch feat-e2e-tests-signout. Closing this PR.

**Pull request closed without merging**